Makefile

```diff
@@ -73,10 +73,16 @@ install: manifests
 
 # Deploy controller in the configured Kubernetes cluster in ~/.kube/config
 .PHONY: deploy
-deploy: manifests $(KUSTOMIZE)
+deploy-via-kustomize: manifests $(KUSTOMIZE)
 	kubectl apply -f config/crd/bases
 	kustomize build config/default | kubectl apply -f -
 
+# Deploy controller to the Kubernetes cluster specified in the environment variable KUBECONFIG
+# Modify the Helm template located at charts/druid/templates if any changes are required
+.PHONY: deploy
```

Comment:
can we rename this to `local-deploy`?

Reply:
The `make deploy` command is designed to be versatile and can be used to deploy `etcd-druid` to any Kubernetes cluster, be it local (like `Minikube` or `KinD`) or remote, as specified in the `KUBECONFIG` environment variable. 

I believe it would be more accurate and less confusing to keep the existing `make deploy` name.

Reply:
Due to https://skaffold.dev/docs/environment/kube-context/ skaffold will also check if there is `$KUBECONFIG` defined and use that instead